exercises/practice/transpose/.docs/instructions.md

```diff
@@ -17,7 +17,8 @@ BE
 CF
 ```
 
-Rows become columns and columns become rows. See <https://en.wikipedia.org/wiki/Transpose>.
+Rows become columns and columns become rows.
+See [transpose][].
```

Comment:
Interesting change... I know it's possible to omit the link text and just show the reference, like `[transpose]`, but I didn't know this was possible. Or is it not and is this a bug? Anyway, let's merge and see.